app/sfr-search-api/swagger.v3.json

```diff
@@ -1400,6 +1519,71 @@
         }
       }
     },
+    "v3EditionDetailObject": {
+      "type": "object",
+      "properties": {
+        "date_modified": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "date_created": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "id": {
+          "type": "integer"
+        },
+        "title": {
+          "type": "string"
+        },
+        "publication_place": {
+          "type": "string"
+        },
+        "publication_date": {
+          "type": "string",
+          "format": "date"
+        },
+        "edition": {
```

Comment:
is this edition id or something different?

Reply:
This is the edition description ("2nd ed" etc.), the `edition_id` is the `id` field a few places up from this.